### DISCUSSION TITLE
Using with full-length scRNA data?

### DISCUSSION THREAD

**Discussion by f6v** [satijalab/seurat#4371] [Q&A] [answered]
I know that the [normalisation procedure](https://doi.org/10.1186/s13059-019-1874-1) implemented in Seurat was described for UMI data. However, is it a good idea to use it for Smart-seq2 data? I'm studying [scCAT data](https://doi.org/10.1038/s41467-018-08205-7) and would like to perform MNN analysis from Seurat 4.

**Comment by saketkc** [comment, marked as answer]
Hi @f6v, as you correctly noted that SCTransform was developed for UMI data. In practice though, it does seem to work well for non-UMI data. See this [issue](https://github.com/ChristophH/sctransform/issues/94#issuecomment-782259403) for more discussion.
